.github/scripts/package.json

```diff
@@ -14,8 +14,8 @@
         "js-yaml": "^4.1.0"
     },
     "devDependencies": {
-        "@octokit/rest": "^21.0.0",
+        "@octokit/rest": "^19.0.0",
         "mocha": "^10.0.0",
-        "sinon": "^20.0.0"
```

Comment:
do these really need to be downgraded?

Reply:
this branch need to be updated, octokit/rest was updated by dependabot last week

Reply:
oh, actually owlbot is rolling back those deps 🤔